src/systems/navsat/NavSat.cc

```diff
@@ -0,0 +1,266 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "NavSat.hh"
+
+#include <ignition/msgs/navsat.pb.h>
+
+#include <string>
+#include <unordered_map>
+#include <utility>
+
+#include <sdf/Sensor.hh>
+
+#include <ignition/common/Profiler.hh>
+#include <ignition/plugin/Register.hh>
+
+#include <ignition/math/Helpers.hh>
+#include <ignition/transport/Node.hh>
+
+#include <ignition/sensors/SensorFactory.hh>
+#include <ignition/sensors/NavSatSensor.hh>
+
+#include "ignition/gazebo/components/NavSat.hh"
+#include "ignition/gazebo/components/LinearVelocity.hh"
+#include "ignition/gazebo/components/Name.hh"
```

Comment:
```suggestion
#include "ignition/gazebo/components/LinearVelocity.hh"
#include "ignition/gazebo/components/Name.hh"
#include "ignition/gazebo/components/NavSat.hh"
```
nit: alphabetical order